department-of-veterans-affairs/abd-vro
# MCP-1620: Fetch claims endpoint

MCP-1620

## Description



### What was the problem?

na

### How does this fix it?

Fetch claims endpoint:  Returns claim submission ID, veteran ICN, and list of contention diagnostic codes for each claim.

### Jira Tickets



- [MCP-1620](https://amida.atlassian.net/browse/MCP-1620)

## How to test this PR

-Make mock data in db, veteran, then claim, then contentions.
-Make sure foreign keys line up (veteranID goes into claim, and claimID goes into contentions)
-Go to swagger and try out the fetch-claims endpoint.
-Verify that claim submission ID, veteran ICN, and list of contention diagnostic codes appear.
                                        

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
